qemu/tests/hotplug_port_chardev_pci_with_console.py

```diff
@@ -0,0 +1,45 @@
+from virttest import error_context
+
+from qemu.tests.virtio_console import add_chardev
+from qemu.tests.virtio_console import add_virtio_ports_to_vm
+from qemu.tests.virtio_serial_file_transfer import transfer_data
+from qemu.tests.virtio_serial_hotplug_port_pci import get_buses_serial_devices
+
+
+def get_virtio_serial_pci(vm, serial_device):
+    serial_device_bus = serial_device.get_param('bus')
+    local_index = serial_device_bus.index('.')
+    serial_bus_id = serial_device_bus[:(local_index)]
+    return vm.devices.get(serial_bus_id)
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Hot-plug chardev and virtserialport
+
+    1. Boot a guest with 1 virtconsole attached to pty backend
+    2. Hot plug unix_socket backend
+    3. Hot add virtserialport, attached on unix chardev
+    4. Transfer data between host and guest via virtserialport
+    5. Hot-unplug existed virt-serial-device, this should fail
```

Comment:
What does this mean? `virtio-serial-pci`? Should fail?

Reply:
Forget to correct it, thanks and update.